src/ripple/basics/impl/IOUAmount.cpp

```diff
@@ -49,6 +51,18 @@ IOUAmount::normalize()
         return;
     }
 
+    if (*stNumberSwitchover)
+    {
+        const Number v{mantissa_, exponent_};
```

Comment:
`const` goes to the east

Reply:
@HowardHinnant , could you please take a look?

Reply:
Fixed: https://github.com/HowardHinnant/rippled/commit/6e1ae23d279c29979d49545be3b5c6169ff8481e